src/data/faq_de.json

```diff
@@ -2003,7 +2003,7 @@
                                     "<b>'URSACHE: 2001 Etwas ist schiefgelaufen. An error occurred while trying to establish a secure connection to the server.'</b>",
                                     "Führen Sie die folgenden Schritte aus, um das Sicherheitszertifikat zu aktivieren. Der genaue Pfad hängt vom Gerät ab.",
                                     "<ol><li>'Einstellungen' -> 'Biometrische Daten und Sicherheit' -> 'Andere Sicherheitseinstellungen' -> 'Sicherheitszertifikate anzeigen' -> 'SYSTEM'</li><li>Scrollen Sie nach unten zu <b>T-Systems Enterprise Services GmbH, T-TeleSec GlobalRoot Class 2</b></li><li>Aktivieren Sie das Zertifikat mit dem Schieberegler</li></ol>",
-                                    "Benutzt Ihr Netzwerk bzw. Ihr Gerät ein Firewall, überprüfen Sie die Erreichbarkeit der Domänen, die in <a href='#access-list' target='_blank' rel='noopener'>[Google/Android]: Firewall/Router Konfiguration: URSACHE: 4000, error during web request, HTTP status 901</a> aufgelistet sind.",
+                                    "Benutzt Ihr Netzwerk bzw. Ihr Gerät ein Firewall, überprüfen Sie die Erreichbarkeit der Domänen, die in <a href='#access-list' target='_blank'>[Google/Android]: Firewall/Router Konfiguration: URSACHE: 4000, error during web request, HTTP status 901</a> aufgelistet sind.",
                                     "Sollten diese Schritte das Problem nicht beheben, schreiben Sie bitte einen Kommentar hier: <a href='https://github.com/corona-warn-app/cwa-app-android/issues/968' target='_blank' rel='noopener'>cwa-app-android/issues/968</a>."

```

Comment:
Should be rel='noopener noreferrer'

Reply:
Fixed in https://github.com/corona-warn-app/cwa-website/pull/3032/commits/2180bce04f3ca6fcfd3f0e9d26b27cae68ebebe1, thanks